src/js/azdata/azdataQueryPlan.js

```diff
@@ -440,7 +464,7 @@ azdataQueryPlan.prototype.placeGraphNodes = function () {
     // for entire showplan. For starters, we set this to 100px. However,
     // if a node has label with many lines, this value will be updated to 
     // better fit that node.
-    this.spacingY = 100;
+    this.spacingY = 105;
```

Comment:
Probably declare a constant variable for this height value as you're using it at multiple places.